tests/new_unit_tests/Commands/Connection/InfoCommandTest.php

```diff
@@ -2,11 +2,13 @@
 
 namespace Pantheon\Terminus\UnitTests\Commands\Connection;
 
-use Consolidation\OutputFormatters\StructuredData\AssociativeList;
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
```

Comment:
The info command uses an AssociativeList, not a RowsOfFields, so the RowsOfFields class does not need to be `use`d here. Seems that neither of these data structures are used directly by this test, though, so the `use AssociativeList` is unnecessary as well.